api/src/pcapi/routes/backoffice/autocomplete.py

```diff
@@ -217,14 +218,14 @@ def prefill_highlights_choices(autocomplete_field: fields.PCTomSelectField) -> N
                 sa_orm.load_only(
                     highlights_models.Highlight.id,
                     highlights_models.Highlight.name,
-                    highlights_models.Highlight.highlight_timespan,
+                    highlights_models.Highlight.highlight_datespan,
                 )
             )
         )
         autocomplete_field.choices = [
             (
                 highlight.id,
-                f"{highlight.name} - {datetime.strftime(highlight.highlight_timespan.lower.date(), '%d/%m/%Y')} - {datetime.strftime(highlight.highlight_timespan.upper.date(), '%d/%m/%Y')}",
+                f"{highlight.name} - {datetime.strftime(highlight.highlight_datespan.lower, '%d/%m/%Y')} - {datetime.strftime(highlight.highlight_datespan.upper, '%d/%m/%Y')}",
```

Comment:
Bug : si je sélectionne une valorisation dans le formulaire de recherche des offres, l"intervalle est bon, mais après avoir cliqué sur « Rechercher », la date de fin passe au lendemain.

Je pense que tu as juste oublié de reporter ici la correction faite ligne 330.
```suggestion
                f"{highlight.name} - {format_datespan(highlight.highlight_datespan)}",
```

Reply:
En effet, merci !